src/lib/dom.js

```diff
@@ -150,6 +150,16 @@ function isTransformableElement(element) {
     return element && (element instanceof Element || element instanceof HTMLElement);
 }
 
+function domRectsAreEqual(a, b) {
+    return a && b
```

Comment:
Whereas semicolon is optional in JavaScript,
a single line `return a && b` may read as `return a && b;`.
Please consider refactoring as follow:
```js
return (
    a && b &&
    a.x === b.x &&
    a.y === b.y &&
    a.top === b.top &&
    a.left === b.left &&
    a.right === b.right &&
    a.bottom === b.bottom
);
```

Reply:
Which also fixes these:
![pull-5302-a](https://user-images.githubusercontent.com/33888540/100383684-39984580-2fec-11eb-9322-7f90747671cc.png)